clients/client-java/src/test/java/com/datastrato/gravitino/client/TestBase.java

```diff
@@ -77,4 +77,8 @@ protected static <T, R> void buildMockResource(
       throws JsonProcessingException {
     buildMockResource(method, path, Collections.emptyMap(), reqBody, respBody, statusCode);
   }
+
+  protected static String withSlash(String path) {
+    return "/" + path;
+  }
```

Comment:
Seems it is only used in test, why do we move here?

Reply:
I moved it to TestBase so we reuse it on the subclass. We can indeed add one on tests.